components/CfvCard.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { TTypeCard } from '../constants';
+
+const CfvCard = ({ title, typeClass }: TTypeCard) => {
```

Comment:
This component is not used, wright ?

Reply:
You are the master of FE :1st_place_medal: